python/tests/catalog/test_rest.py

```diff
@@ -64,14 +64,9 @@ def rest_mock(requests_mock: Mocker):
 
 
 def test_no_uri_supplied():
-    with pytest.raises(ValueError) as exc_info:
+    with pytest.raises(KeyError):
         RestCatalog("production")
 
-    assert (
```

Comment:
This is okay with me, this is also covered by `tests.cli.test_console:test_missing_uri` which is user-facing.

Reply:
Yeah, that's what I thought, too. Best to test it there.